LoginMetricsTest/UnitTests.cs

```diff
@@ -0,0 +1,99 @@
+using System;
+using System.Collections.Generic;
+using Xunit;
+using LoginMetricsMockData;
+using LoginMetrics;
+using LoginMetricsInterfaces;
+
+namespace LoginMetricsTest
+{
+    public class UnitTests
+    {
+        ILoginMetricsRepo _testRepo;
+        public UnitTests() {
+            _testRepo = new MockRepo(); //is read only, so stateless with respect to tests
+        }
+
+        void CheckAnswerEqualsResult(List<Metric> answer, List<Metric> result) {
+            Assert.Equal(answer.Count, result.Count);
+            for (var i = 0; i < answer.Count; i++)
+            {
+                Assert.True(answer[i].Subject == result[i].Subject);
+                Assert.True(answer[i].Value == result[i].Value);
+            }
+        }
+
+        [Fact]
+        public void Test_Top2LoginsInOctober()
+        {
+            //Define answer
+            var top = new Metric("Alice", 13);
+            var next = new Metric("Charlie", 7);
+            var answer = new List<Metric>();
+            answer.Add(top);
+            answer.Add(next);
```

Comment:
```suggestion
var answer = new List<Metric> { 
   new Metric("Alice, 13),
   new Metric("Charlie", 7)
};
```

pros/cons to this suggestion?

Reply:
I'd say both pros and cons are minor. The inline initialization is maybe more readable.